hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/Region.java

```diff
@@ -92,7 +92,7 @@
    * <p>Use with caution.  Exposed for use of fixup utilities.
    * @return a list of the Stores managed by this region
    */
-  List<? extends Store> getStores();
+  Collection<? extends Store> getStores();
```

Comment:
Region is exposed to CPs.  
Also that is the reason why direct internal data structure is not exposed I believe.

Reply:
Whoops, I accidentally looked at HRegion instead of Region when reviewing the IA. Since Region is exposed to CPs, I guess it's not safe to make this change.

Reply:
Pls remove this change completely. This is an exposed interface

Reply:
Yeah, I forgot to mention this is a breaking change, and was wondering if it would be acceptable for HBase 3 (master branch) only.

It might not worth the effort, though.